src/components/PatchesPage/index.tsx

```diff
@@ -139,16 +137,20 @@ export const getPatchesInputFromURLSearch = (search: string): PatchesInput => {
   };
 };
 
-const FlexRow = styled.div`
-  display: flex;
-  flex-grow: 2;
-`;
 const PaginationRow = styled.div`
   display: flex;
   justify-content: flex-end;
   align-items: center;
 `;
 
 const FiltersWrapperSpaceBetween = styled(FiltersWrapper)`
-  justify-content: space-between;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr) 2fr;
+  grid-template-rows: 1fr;
+  grid-column-gap: ${size.s};
+  grid-row-gap: 0px;
+`;
```

Comment:
Seems like this results in a pretty different layout on the My Patches page, namely that the Checkbox is no longer aligned to the right. Is that intentional? :0

on prod:

<img width="1473" alt="Screen Shot 2022-04-15 at 11 31 27 AM" src="https://user-images.githubusercontent.com/47064971/163590097-3edf97d7-d47c-4c18-9175-d66e0d84c8f8.png">

on this PR:

<img width="1462" alt="Screen Shot 2022-04-15 at 11 31 37 AM" src="https://user-images.githubusercontent.com/47064971/163590113-be986604-c8c5-4e7f-bf3a-2c4480b299a1.png">

Reply:
I just updated it
<img width="1680" alt="Screen Shot 2022-04-15 at 4 13 09 PM" src="https://user-images.githubusercontent.com/10734386/163634037-2b818e24-6394-47fe-8674-162a797b2b0f.png">